openpype/hosts/houdini/plugins/publish/validate_primitive_hierarchy_paths.py

```diff
@@ -104,3 +112,47 @@ def get_invalid(cls, instance):
                 "(%s of %s prims)" % (path_attr, len(invalid_prims), num_prims)
             )
             return [output_node.path()]
+
+    @classmethod
+    def repair(cls, instance):
+        """Repair action if `Path Attribute` wasn't exist.
+
+        It Adds a "name" node before the the output node to automatically
+        force a path attribute on the geometry. Be aware that this will
+        create a `default single name` and thus will merge all geometry in one shape.
+        """
+
+        output_node = instance.data.get("output_node")
+        rop_node = hou.node(instance.data["instance_node"])
+
+        if not (isinstance(output_node, hou.SopNode) and \
+                 output_node.type().name() == "output"):
```

Comment:
I think the assumption that it'll be an output node is just not 100% accurate - the export works fine even if it's not an output node.